xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -597,7 +597,7 @@ void CGUIDialogVideoInfo::Play(bool resume)
     return;
   }
 
-  CFileItem movie(*m_movieItem->GetVideoInfoTag());
+  CFileItem movie(*m_movieItem);
```

Comment:
i know for a fact that 5 years ago i tracked a bug down to the fact that we did what you suggest, and changed it to what it is now. i cannot guarantee whether the problem still exists but this potentially causes regressions and i this needs to be investigated! i do not recall the details..